src/main/scala/com/github/traviscrawford/spark/dynamodb/DynamoBackupJob.scala

```diff
@@ -36,7 +43,14 @@ object DynamoBackupJob extends Job {
       case false => None
     }
 
+    if (overwritePath()) deleteOutputPath(output())
+
     DynamoScanner(sc, table(), totalSegments(), pageSize(),
       maybeCredentials, maybeRateLimit, maybeRegion).saveAsTextFile(output())
   }
+
+  def deleteOutputPath(output: String): Unit = {
```

Comment:
Please also log a message saying which path is being deleted, this would be very useful in debugging.